lms/services/auto_grading.py

```diff
@@ -0,0 +1,54 @@
+from lms.js_config_types import AnnotationMetrics
+from lms.models import AutoGradingConfig
+
+
+def calculate_grade(
+    auto_grading_config: AutoGradingConfig, annotation_metrics: AnnotationMetrics
+) -> float:
+    """Calculate auto grades based on the config and the number of annotations made.
+
+    The results is a 100 based float rounded to two decimals.
+    """
+    combined_count = annotation_metrics["annotations"] + annotation_metrics["replies"]
+
+    grade: float = 0
+    match (auto_grading_config.activity_calculation, auto_grading_config.grading_type):
```

Comment:
This, and the test cases follow the examples on the [PRD](https://docs.google.com/document/d/1lK8Wn2n4Cr7UI7mwaSDBun4R2aUH-P7EZl-x8Z9RCxY/edit)